packages/block-library/src/site-logo/edit.js

```diff
@@ -337,6 +337,7 @@ export default function LogoEdit( {
 	);
 
 	const classes = classnames( className, {
+		'is-default-size': ! width,
 		'is-resized': !! width,
```

Comment:
Do we still need this then? Seems like there's only one occurence left here: https://github.com/WordPress/gutenberg/blob/82ddc9e405d91066eba6d08fb716ef8feea922ab/packages/block-library/src/site-logo/editor.scss#L13-L15

...which we could probably replace with a  `&:not(.is-default-size)`?

Reply:
The is-resized rule, and the table display, was present before the changes I made to default sizes, and I believe are still necessary. I'd prefer to not touch them, or at least touch them separately if need be.

_I suspect they were ported over when the block was copied from its Image block roots, and in the image block the table property is used to size the image the same as the caption when manually sized. But I'd still like to fix that separately._

Reply:
Oh, I didn't mean to remove the rule entirely, but rather to change its selector to `&:not(.is-default-size)` so we can drop the now-redundant `is-resized` className 😄

Reply:
(If we can't do that, I think I'd prefer #30845, since it doesn't introduce another class name that is essentially just `is-resized` negated, which is kinda redundant.)

Reply:
Oh of course! I'll get on it. Can I have 20 minutes? I have to pick up a kid very quick

Reply:
Sure! 😄

Reply:
Okay here we go! 
<img width="855" alt="Screenshot 2021-04-15 at 13 16 31" src="https://user-images.githubusercontent.com/1204802/114860918-dec18800-9dec-11eb-96c4-818d0b8436d6.png">
![Uploading Screenshot 2021-04-15 at 13.16.37.png…]()

Seemes to work fine! Thank you Bernie!